data/remnant/remnant 1 introduction.txt

```diff
@@ -169,47 +156,15 @@ mission "Remnant: Lost in Ember Waste 1"
 
 
 
-mission "Remnant: Lost in Ember Waste Compatibility"
-	landing
-	invisible
-	source
-		government "Remnant"
-	to offer
-		has "lost in ember waste: declined"
-		not "Remnant: Lost in Ember Waste: declined"
-	on offer
-		set "Remnant: Lost in Ember Waste: offered"
-		set "Remnant: Lost in Ember Waste: declined"
-		set "event: remnant: lost in ember waste"
-		fail
-
-
-
-mission "Remnant: Lost in Ember Waste 2 Compatibility"
+mission "Remnant: Lost in Ember Waste"
 	landing
 	invisible
 	source
 		government "Remnant"
 	to offer
-		has "lost in ember waste: declined"
-		has "Remnant: Lost in Ember Waste: declined"
-	on offer
-		set "Remnant: Lost in Ember Waste: offered"
-		set "Remnant: Lost in Ember Waste: declined"
-		set "Remnant: Lost in Ember Waste 2: offered"
-		set "Remnant: Lost in Ember Waste 2: declined"
-		set "event: remnant: lost in ember waste"
-		fail
-
-
-
-mission "Remnant: Lost in Ember Waste 2"
-	landing
-	invisible
-	source
-		government "Remnant"
-	to offer
-		has "event: remnant: lost in ember waste"
+		or
+			has "event: remnant: lost in ember waste"
+			has "event: lost in ember waste"
```

Comment:
And if the trigger mission can now set itself again, we don't need to cover both event names.
```suggestion
		has "event: remnant: lost in ember waste"
```